3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.Android/MainActivity.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+
+using Android.App;
+using Android.Content.PM;
+using Android.Runtime;
+using Android.OS;
+using activedirectoryxamarinintune;
+using Android.Content;
+using Microsoft.Identity.Client;
+
+namespace active_directory_xamarin_intune.Droid
+{
+    [Activity(Label = "Xamarin Intune Sample", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize)]
+    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    {
+        private const string AndroidRedirectURI = "msauth://com.yourcompany.xamarinintuneapp/JjmT52ASxa1Lz55s+qPPgxb5xeo="; // TODO - Replace with your redirectURI
```

Comment:
This redirect URI is a critical part to get right. we need to be 100% clear on how to acquire it. Wouldnt be a bad idea to link to the MS Docs on this since those docs are more stable